src/nyc_trees/apps/survey/management/commands/send_reminder_emails.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import smtplib
+
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from django.utils.timezone import now
+
+from apps.core.models import TaskRun
+from apps.survey.models import BlockfaceReservation
+from apps.mail.views import send_reservation_reminder
+
+_COMMAND_NAME = 'apps.survey.management.commands.send_reminder_emails'
+_ALREADY_SENT_MESSAGE = ('WARNING: Reservation emails sent already today. '
+                         'Doing nothing!')
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        today = now().date()
+        _, unlocked = TaskRun.objects.get_or_create(name=_COMMAND_NAME,
+                                                    date_started=today)
```

Comment:
Because we are tracking successful notifications directly on the reservation row, we don't need to enforce that this command runs only once per day, just that it does not run more than once at any given time.

If it makes things easier to enforce the once per day rule, that is fine, and we don't have to rip out an already working thing. In the unlikely event that we would ever want to force it to run more than once per day, we could manipulate the taskrun table.